### ISSUE TITLE
Authoring dashboard

### ISSUE THREAD

**Issue by knownasilya** [stonecircle/ember-ghost#2]
I'd love to see something like the Ghost Admin interface for this, with a simpler focus and authoring markdown to github.

Ghost is using the new koenig editor, which they have as an addon https://github.com/TryGhost/Ghost-Admin/tree/master/lib/koenig-editor so maybe that can be a starting point.

**Comment by JonForest** [comment]
How would we do this?  Let's assume the editor was available to everyone if the app was built in dev mode, since we don't have users/roles.  The app still couldn't write to disk because of the permissions that a web app has.

Would it therefore only make sense as an accompanying electron app? 
Or could we work in the Fastboot realm to get server permissions and actually write the file via our 'server'?  I'm not at all familiar with fastboot, but this feels like it could be possible.
Finally, since `ember s` is actually launching an express instance, I wonder if we could do something in the ember-cli to add a 'save' end-point?

I'm operating a little beyond the edge of my ember knowledge here, so forgive me if any of these suggestions are patently absurd.

**Comment by knownasilya** [comment]
I could see the dashboard being the dummy app and running ember-cli-deploy to github pages on publish.

**Comment by JonForest** [comment]
Github pages feels a little hosting specific, and I'm holding my repo in GitLab, so GitHub APIs are not that helpful either.
There might be a way of dealing with the variety of git targets in config however?

**Comment by knownasilya** [comment]
Yeah for sure. The other thing I want to do is do a p2p version where you can self-publish via dat/beakerbrowser, but that is drastically different from a regular implementation.
